chain-core/src/init/address.rs

```diff
@@ -135,6 +145,25 @@ impl RedeemAddress {
     }
 }
 
+impl CroAddress<RedeemAddress> for RedeemAddress {
+    fn to_cro(&self) -> Result<String, ()> {
+        let checked_data: Vec<u5> = self.0.to_vec().to_base32();
+        Bech32::new("crms".into(), checked_data)
+            .map_err(|_e| ())
+            .and_then(|b| {
+                let bytes = b.to_string().as_bytes().to_vec();
+                String::from_utf8(bytes).map_err(|_e| ())
+            })
+    }
+
+    fn from_cro(encoded: &str) -> Result<Self, ()> {
+        encoded
+            .parse::<Bech32>()
+            .map_err(|_e| ())
```

Comment:
for parsing, there could be many possible errors: https://docs.rs/bech32/0.6.0/bech32/enum.Error.html

perhaps create an enum error type that would list / wrap all the possible errors:
```
enum CroAddressError {
   Bech32Error(bech32::Error),
   ...
}
```
and this could be returned?

Reply:
ok